ignite/pkg/availableport/availableport_test.go

```diff
@@ -0,0 +1,83 @@
+package availableport_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/ignite/cli/ignite/pkg/availableport"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAvailablePort(t *testing.T) {
+
+	ports, err := availableport.Find(10)
+	require.Equal(t, nil, err)
+	require.Equal(t, 10, len(ports))
+
+	for idx := 0; idx < 9; idx++ {
+		for jdx := idx + 1; jdx < 10; jdx++ {
+			require.NotEqual(t, ports[idx], ports[jdx])
+		}
+	}
+
+	// Case no ports generated
+	options := availableport.OptionalParameters{
+		WithMinPort: 1,
+		WithMaxPort: 1,
+	}
+	ports, err = availableport.Find(10, options)
```

Comment:
You should use options using something like:

```suggestion
	options := []availableport.Options{
		availableport.WithMinPort(1),
		availableport.WithMaxPort(1),
	}
	ports, err = availableport.Find(10, options...)
```